components/home/OdometerCounter.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="odometer-counter">
+    <template v-if="numStr !== '0'">
```

Comment:
would it be possible that `numStr` is falsy or negative?

Reply:
updated